content/networking/security-groups.md

```diff
@@ -1,70 +1,43 @@
 /*
 Title: Security Groups
+Sort: 0
 */
 
-By default, a raw instance only has port 22 open so you can SSH in after it's
-created. We don't want to expose your servers to anything a brand new instance
-isn't prepared for. To manage access, we use Security Groups. The best practise 
-for security groups, is to make them as specific as possible. A single instance
-can belong to multiple security groups, so for a web server, you might want
-ports 22, 443, and 80 open. Or for a DB server you might want port 5432 open
-for PostgreSQL.
+Network Security Groups allow you to open traffic on a port-by-port basis. 
+This flexible model can extend to create any number of situations, whether
+simple or complex while maintaining strict security.
 
-If access is attempted to a port/IP combination that is not assigned, no
-connection will be established at all and the connection will time out.
+## Restricted by Default
 
-The default security group has just port 22 open, so we'll create a brand new
-one named "Web" for a web server as an example.
+By default, a raw instance only has port 22 open so you can SSH in after 
+it's created. We don't want to expose your servers to anything a brand new 
+instance isn't prepared for. 
 
-## Creating a "Web" Security Group
+## Closed Port Behaviour
 
-Navigate to "Access & Security", under the "Servers" menu, then click "create
-security group". Fill in your desired identifier and description for the new
-security group. Note that your identifier can only contain upper or lower case
-letters (no spaces or symbols).
+If access is attempted to a port/IP combination that is not permitted, no
+connection will be established at all and the connection will time out when the
+connecting client terminates.
 
+## Best Practises
 
-![Create Securit Group](/img/content/networking/web-security-group.png)
+The best practise for creating your security groups is to make them as 
+**specific** to a use case as possible in order to retain flexibility. The
+screenshot below shows an example of use-case groups being assigned to a server.
 
-Then, edit your security group's rules to add the required ports by clicking
-"Edit Rules".
+![Security Group Assignment](/img/content/networking/security-group-multi-assign.png)
 
-![Web Security Group](/img/content/networking/security-group-list.png)
+For example, one might create individual Security groups for SSH access, Web 
+Server ports, DB server ports, and VPN access. A single instance (even 
+individual ports on the same server) can belong to multiple security groups. 
```

Comment:
Nitpick: instances don't "belong" to security groups, security group rules do :). I would use "associated" instead